crates/hir-ty/src/consteval.rs

```diff
@@ -287,7 +287,7 @@ pub(crate) fn const_eval_discriminant_variant(
     }
 
     let repr = db.enum_data(loc.parent).repr;
-    let is_signed = IsNoneOr::is_none_or(repr.and_then(|repr| repr.int), |int| int.is_signed());
+    let is_signed = Option::is_none_or(repr.and_then(|repr| repr.int), |int| int.is_signed());
```

Comment:
Let's use the method syntax (`opt.is_none_or(f)` instead of `Option::is_none_or(opt, f)`.

Reply:
yeah sorry !